docs/src/main/asciidoc/mailer-reference.adoc

```diff
@@ -673,6 +673,27 @@ quarkus.mailer.tls=true
 quarkus.mailer.mock=false # In dev mode, prevent from using the mock SMTP server
 ----
 
+=== Office 365
+
+==== Prerequisites
+
+1. Enable SMTP Access to your Office 365 mailbox, you can do it from the administration console.
```

Comment:
It would be nice to have a link to the instructions too, something like:

```suggestion
1. Enable SMTP Access to your Office 365 mailbox, you can do it from the administration console (see [this guide](https://learn.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission) for more information)
```